source/docs/articles/sites/redis-as-a-caching-backend.md

```diff
@@ -163,6 +165,44 @@ To check the # of keys in the cache, you can use the `DBSIZE` command. The follo
 
 ## Troubleshooting
 
+### Redis is enabled but doesn't have any data
+
+When the Dashboard Status check reports that Redis is enabled but doesn't have any data (0 keys found), you'll want to confirm the logic behind the check for PANTHEON_ENVIRONMENT in your `settings.php` Redis cache configuration. Depending on the kind of test you're performing, you’ll get different results. Here is an example of a block that will result in an incorrectly configured cache backend:

```

Comment:
Sorry, should have commented here. The last line in this paragraph should be bold in order to ensure readers don't see the code snippet and grab without understanding it is an example of an *incorrect* code block.